codewof/templates/users/user_detail.html

```diff
@@ -0,0 +1,48 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block title %}User: {{ user.get_full_name }}{% endblock %}
+
+{% block content %}
+  <div class="container">
+
+    <div class="row">
+      <div class="col-sm-12">
+        <h1>{{ user.get_full_name }}</h1>
+
+        <p><strong>Date joined</strong>: {{ user.date_joined }}</p>
+        <p><strong>Points earned</strong>: {{user.profile.points}}</p>
+        <b>Achievements:</b>
+        <br>
+        {% for badge in all_badges %}
+            {% if badge in user.profile.earned_badges.all %}
+              <img src="{% static badge.icon_name %}">
+              <u>{{badge.display_name}}</u>
+            {% else %}
+            <i>{{badge.display_name}}</i>
```

Comment:
![image](https://user-images.githubusercontent.com/25916475/65398654-9c041d80-de0c-11e9-8f5e-fc3cd8350269.png)

Perhaps only show the "best" badge of the set at once, i.e. the 5 questions one and not the 1 question one in this case